pkg/server/plugin/nodeattestor/devid/challenge.go

```diff
@@ -0,0 +1,99 @@
+// (C) Copyright 2021 Hewlett Packard Enterprise Development LP
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License. You may obtain
+// a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations
+// under the License.
```

Comment:
I think we need to remove this license header as the code is being contributed to the project and will fall under CNCF copyright... I think.

Reply:
Sure! This implementation is based on some internal work we did. I forgot to remove the license header 🤦🏻‍♂️